templates/fine-tune-llm_v2/README.md

```diff
@@ -85,4 +78,56 @@ You can edit the values, such as `context_length`, `num_epoch`, `train_batch_siz
 In addition, the deepspeed configs are provided in case you would
 like to customize them.
 
+### What's the full list of supported models?
+
+The following models can currently be fine-tuned with this template.
```

Comment:
Give a general guideline, say in general any model that is compatible with the architecture of the mentioned base models can be finetuned for both LoRA and full parameter using the same configs as the base models. 

This is a growing list but it includes the following models: